helpers/lib_tasks_pytest.py

```diff
@@ -864,6 +865,8 @@ def run_coverage_report(  # type: ignore
         if exclude_from_report is not None:
             report_html_cmd += f" --omit={exclude_from_report}"
         report_cmd.append(report_html_cmd)
+    # Generate an XML report for Codecov.
+    report_cmd.append("coverage xml -o coverage.xml")
```

Comment:
Can we disable these comments

<img width="387" alt="image" src="https://github.com/user-attachments/assets/a403c3c4-b571-43be-83ba-0b2470a5bbf0" />

 from Codecov? 

I think they are cool, but for now they add a lot of noise.